Cargo.toml

```
@@ -10,8 +10,12 @@ authors = ["MaidSafe Developers <dev@maidsafe.net>"]
 edition = "2018"
 
 [dependencies]
-threshold_crypto = "~0.3.1"
+hex = "~0.3.1"
+maidsafe_utilities = "~0.17.0"
```

We published 0.18.0 recently, so could you use it here too please?
```suggestion
maidsafe_utilities = "~0.18.0"
```